app/static/js/PointsDrawer.js

```diff
@@ -13,6 +14,17 @@ function PointsDrawer(canvas) {
     this.width = canvas.node().width;
     this.height = canvas.node().height;
     this.context = canvas.node().getContext('2d');
+
+    this.overlay = 'None';
+
+    // default colourscale
+    // this colourscale is used when no overlay is active
+    this.defaultColourScale = d3.scale.linear()
+        .domain([-3, 3])
```

Comment:
Ahh that call to domain doesn't need to be there actually! The scale will map to the default colour whatever you feed into it..